docker-hub/onboarding-faqs.md

```diff
@@ -29,7 +29,7 @@ manage users, teams, and repositories in the organization. [Learn more](orgs.md)
 
 ### How many organizations can I create?
 
-You can create multiple organizations. However, if you’re enabling SSO and have multiple organizations, each organization must have a domain associated with it.
+You can create multiple organizations. You must first set up your first organization and contact the Customer Success team at customer-success@docker.com to add the additional organizations. Also, if you’re enabling SSO and have multiple organizations, each organization must have a domain associated with it.
```

Comment:
Would recommend you "should" instead of "must" first set up

For the last sentence, could we change it to: "If you are enabling SSO, it will be configured based on domain not your organizations."

Reply:
You must first set up your first organization--> `you must set up your first organization`
(removing one first in the sentence)